### PR TITLE
AddCar and SignUp mocks added

### PR DESCRIPTION
**Pull request by CIsbis**

### CODE REVIEW

File: src/SignIn_mock.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        sign_up_instance.sign_up()
+
+        expected_row = ['test@example.com', 'John', 'Doe', 'password123']
+        mock_csv_writer_instance = mock_csv_writer.return_value
+        mock_csv_writer_instance.writerow.assert_called_once_with(expected_row)
+
+        self.assertEqual(mock_input.call_count, 4)  # Ensure input is called 4 times
+        self.assertEqual(sign_up_instance.sign_up(), "Signed up successfully!")
+
```
Comment (shannondesiree) [inline src/SignIn_mock.py:22]: I would consider adding more test cases to cover edge cases, error handling, and boundary conditions but a good start

Comment (careym04) [reply src/SignIn_mock.py:22]: looks good, could consider using stubbed data from a config interface for the testing
